### PR TITLE
Prettify inferred infix types #685

### PR DESCRIPTION
**Pull request by oyvindberg**

### CODE REVIEW

**Comment by oyvindberg** [comment]
Alright! How is it looking, @olafurpg ?

File: scalafix-tests/output/src/main/scala/test/explicitResultTypes/ExplicitResultTypesInfix.scala [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+import test.explicitResultTypes.{ ExplicitResultTypesInfix, \/ }
+trait \/[A, B]
+
+object ExplicitResultTypesInfix {
+  val ab: Int \/ String = null.asInstanceOf[Int \/ String]
+  def foo: Int \/ String = ab
+
+  case class :+:[T1, T2]()
+  def bar: ExplicitResultTypesInfix.:+:[Double, Long] = :+:[Double, Long]()
```
Comment (oyvindberg) [inline scalafix-tests/output/src/main/scala/test/explicitResultTypes/ExplicitResultTypesInfix.scala]: btw is this the best we can do for now? this type has a prefix, but it's unnecessary in this context, heh

Comment (oyvindberg) [reply scalafix-tests/output/src/main/scala/test/explicitResultTypes/ExplicitResultTypesInfix.scala]: @olafurpg

**Comment by fommil** [comment]
thank you so much!

**Pull request merged**